packages/wallet/test/integration/cip30mapping.test.ts

```diff
@@ -270,7 +270,7 @@ describe('cip30', () => {
       });
       test('throws an error when there are no UTxOs in the wallet', async () => {
```

Comment:
```suggestion
      test('returns null when there are no "unspendable" UTxOs in the wallet', async () => {
```

Reply:
Fixed in [ca144bd](https://github.com/input-output-hk/cardano-js-sdk/pull/602/commits/ca144bd8b8d13c7a628d493ed3dcf073898c4129)